engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/ejea/EjeaFetchingCachedOutputsFromDatabaseSpec.scala

```diff
@@ -29,9 +31,11 @@ class EjeaFetchingCachedOutputsFromDatabaseSpec extends EngineJobExecutionActorS
         ejhaResponse foreach { ejea ! _ }
 
         // Send the response from the "Fetch" actor
-        val cachedResults = successResponse.jobOutputs
-        ejea ! CachedOutputLookupSucceeded(cachedResults, CacheHit(MetaInfoId(75)))
-        helper.callCacheHitCopyingProbe.expectMsg(CopyOutputsCommand(cachedResults))
+        val cachedSimpletons = Seq(WdlValueSimpleton("a", WdlString("hullo")), WdlValueSimpleton("b", WdlString("cheerio")))
+        val detritusMap = Map("stdout" -> "//somePath")
+        val cachedReturnCode = Some(17)
+        ejea ! CachedOutputLookupSucceeded(cachedSimpletons, detritusMap, cachedReturnCode, CacheHit(MetaInfoId(75)))
+        helper.callCacheHitCopyingProbe.expectMsg(CopyOutputsCommand(cachedSimpletons, detritusMap, cachedReturnCode))
```

Comment:
Excellent work. Really neat, concise, wonderful code. A thousand bravo's to the author!